package.json

```diff
@@ -14,21 +14,27 @@
   ],
   "license": "MIT",
   "main": "index.js",
+  "exports": {
+    "require": "./index.js",
+    "import": "./.esm-wrapper.mjs"
+  },
   "repository": {
     "type": "git",
     "url": "https://github.com/pkgjs/nv.git"
   },
   "scripts": {
     "test": "standard && mocha",
     "debug": "mocha --inspect --inspect-brk --timeout=0",
-    "prepublushOnly": "npm t",
-    "postpublish": "git push origin && git push origin --tags"
+    "prepublishOnly": "npm t",
+    "postpublish": "git push origin && git push origin --tags",
+    "prepack": "gen-esm-wrapper . ./.esm-wrapper.mjs"
```

Comment:
I wonder if tracking this file in git would be good?  It might make debugging changes more easy since a change in the what the underlying package writes will not have a visible change in the repo.

Reply:
Sure, I personally don’t do it because in the end it’s a build artifact, but if you think it makes sense, I’d be happy to remove it (i.e. just say what you prefer :slightly_smiling_face:)

Reply:
I am not sure how I feel, maybe some other folks on @pkgjs/nv-collaborators have an opinion.

Reply:
If I understood well this example https://github.com/addaleax/gen-esm-wrapper#example-project outputs to `dist` which is something common to be added in gitignore. So I think that makes sense to add to gitignore as well :+1: